src/views/Home.vue

```diff
@@ -8,7 +8,7 @@
       :perPage="5"
     />
     <ThanksForSuggesting
-      :NoResultsToShow="noResults"
+      v-if="noResults"
```

Comment:
Another `v-if` that prevents rendering the component with errors when it doesn't have the data it needs.